src/lib/components/add-portfolio.svelte

```diff
@@ -45,6 +45,7 @@
 			start_date: startDate.toDateString(),
 			end_date: endDate.toDateString(),
 			inflation_rate: inflation,
+			link: null,
```

Comment:
To stay consistent :)
```suggestion
			link: undefined,
```

Reply:
I would be happy not using `null`, but unfortunately in SQL there is a `NULL` value indicating a missing value and the Supabase type generator converts that to Typescript `null` value.

See here:
https://github.com/snaha/kalkul/pull/254/files#diff-bffd5e4300c7d550a0d30d4c443baf638b9e493bc96278ef30f71ae40f9b74d7R151